features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher-new/source/src/app/components/Apis/Details/Monetization/BusinessPlans.jsx

```diff
@@ -0,0 +1,151 @@
+            const filteredPolicies = policies.filter(policy => policy.tierPlan === 'COMMERCIAL');
+            this.setState({ policies: filteredPolicies });
+        });
+        api.getMonetization(api.id).then((status) => {
+            this.setState({ monetizedPolices: status.properties });
+        });
+    }
+
+    monetizationQuery(policyName) {
+        const { monetizedPolices } = this.state;
+        return policyName in monetizedPolices;
+    }
+
+    /**
+     *
+     * @inheritdoc
+     * @returns {React.Component} Policies / Business plans list
+     * @memberof BusinessPlans
+     */
+    render() {
+        const { policies, monetizedPolices } = this.state;
+        if (monetizedPolices === null) {
+            return <Progress />;
+        }
+        const policiesList = policies.map(policy => (
+            <Grid item xs={6} spacing={2}>
+                <FormControlLabel
+                    control={
+                        <Checkbox id='monetizationStatus' checked={this.monetizationQuery(policy.name)} />
+                    }
+                    label={policy.name}
+                />
+                {
+                    Object.keys(policy.monetizationAttributes).map((key) => {
+                        if (policy.monetizationAttributes[key] !== null) {
+                            return (
+                                <Typography component='p' variant='body1'>
+                                    { key } : {policy.monetizationAttributes[key] }
+                                </Typography>
+                            );
+                        } else {
+                            return false;
+                        }
+                    })
+                }
+            </Grid>
+        ));
+        return (
+            <Grid container className={classes.root}>
+                <Grid className={classes.grid} spacing={2}>
+                    <Grid>
+                        <Typography variant='subtitle' gutterBottom>
+                            <FormattedMessage id='commercial.policies' defaultMessage='Commercial Policies' />
+                        </Typography>
+                    </Grid>
+                    {
+                        (policies.length > 0) ? (
+                            <Grid>
+                                <Typography>
+                                    <FormattedMessage
+                                        id='unchecked.policies.are.not.monetized.click.save.to.monetize'
+                                        defaultMessage='Unchecked polices are not monetized, click save to monetize'
+                                    />
+                                </Typography>
+                            </Grid>
+                        ) : (
+                            <Grid>
+                                <Typography>
+                                    <FormattedMessage
+                                        id='no.commercial.policies.to.monetize'
```

Comment:
Hi @shilmyhasan 
Please follow the i18n id naming convention mentioned in [here](https://github.com/wso2/carbon-apimgt/wiki/How-internationalization-(i18n)-works-in-API-Manager-React-Apps#id-naming-convention).

Reply:
added in b29eb6f